app/Http/Controllers/MessagesController.php

```diff
@@ -92,7 +92,9 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        // $competition = null;
+
+        $contest = Contest::find(request('contest_id'));
+        $this->manager->appendCampaign($contest);
```

Comment:
it appends it sure, but it doesn't get actually added to the `$contest` unless you reassign `$contest` to the return value from `$this->manager->appendCampaign($contest);`